### DISCUSSION TITLE
Pull-down list opening upwards instead of downwards

### DISCUSSION THREAD

**Discussion by wduda** [wduda/TravelWindowII#243] [Feature Ideas & Requests]
>Would it be possible to add an option to make the mini-window with the pull-down list open upwards instead of downwards?
> Right now i can't position the travel window where i want it on the screen because a part of the list will appear below the screen border.

**Comment by wduda** [comment]
Would it be preferable to have this configureable or that it would automatically switch direction to best utilise screen space?

**Reply by whiterabbit963** [reply]
I think doing a screen check and automatically opening upwards would be simple enough.

**Reply by wduda** [reply]
I tried checking the code but it seems this is actually based on ComboBox which seems to not have that kind of functionality already. Will look into that more.

**Reply by whiterabbit963** [reply]
My thought was that we should be able to redraw with new starting coordinates. Of course, the initial idea rarely works out as advertised :)
Edit: it does look this easy. I am looking at an implementation and will test it later today when I get some more free time.

**Comment by wduda** [comment]
Will be resolved in #244 scheduled for v2.5.1
